docs/en/latest/plugins/response-rewrite.md

```diff
@@ -118,6 +118,71 @@ X-Server-balancer_addr: 127.0.0.1:80
 {"code":"ok","message":"new json body"}
 ```
 
+Replace the string X-Amzn-Trace-Id of body to X-Amzn-Trace-Id-Replace.
```

Comment:
Hi, the changes you added are more like a scenario section (not related to the `Enabling` section), please see https://github.com/apache/apisix/blob/a729489152a032571b9bcd9b19edc75949755fbe/docs/en/latest/plugins/response-rewrite.md

![image](https://user-images.githubusercontent.com/2106987/172291030-8f35e5a2-c0d3-4e67-bd9a-d7c99299db48.png)

cc @navendu-pottekkat should we accept this? If yes, then what titles should we use?

Reply:
This could be added as an example. But it is not clear to me that we are replacing the header using regex.

Reply:
I add description let it clear.